runtime-attach/build.gradle.kts

```diff
@@ -11,4 +11,15 @@ dependencies {
 
   // Used by byte-buddy but not brought in as a transitive dependency.
   compileOnly("com.google.code.findbugs:annotations")
+
+  testImplementation("io.opentelemetry.javaagent:opentelemetry-javaagent:1.14.0")
```

Comment:
can you move this version to dependency management section so we don't miss updating it?

Reply:
Done